dashboard/test/ui/README.md

```diff
@@ -38,7 +38,7 @@ Sauce Labs requires credentials to be set in locals.yml first.
 # code-dot-org/locals.yml
 saucelabs_username: 'yourusername'
 saucelabs_authkey: 'xxxxxx-xxxx-xxxx-xxx-xxxxxxxxx'
-
+saucelabs_tunnel_name: cdo-tunnel
```

Comment:
reduce decisions: suggest a default tunnel name, use the same name we use on drone